README.md

```diff
@@ -143,6 +143,8 @@ plaidClient.getCategories(cb);
 plaidClient.resetLogin(access_token, cb);
 // Sandbox-only endpoint to trigger a webhook for an Item
 plaidClient.sandboxItemFireWebhook(access_token, webhook_code, cb);
+// Sandbox-only endpoint to set the verfication_status for an Item
+plaidClient.sandboxItemSetVerificationStatus(access_token, webhook_code, cb);
```

Comment:
Copy-paste error? The parameters here look like they were copied from `sandboxItemFireWebhook` above.